DM-2434 Updated "How does the platform support developing agents?" section. (#1193)
SoftwareAG/c8y-docs: content/concepts/interfacing-devices-bundle/developing-agents.md

{{< product-c8y-iot >}} supports agent development on three different levels:

> * It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/)

Reviewer: ```suggestion
* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/).
```

{{< product-c8y-iot >}} supports agent development on three different levels:

> * There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories, e.g. [Cumulocity IoT Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) in Python.

Reviewer: ```suggestion
* There are a number of full-featured open source agents and drivers in {{< company-c8y >}}'s [github.com](https://github.com/SoftwareAG/cumulocity-examples) and [mbed.org](http://mbed.org/users/vwochnik/code/) repositories, for example, the [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) in Python.
```
